Scripts/datatypes/purpose.py

```diff
@@ -73,6 +73,11 @@ def zone_numbers(self):
     @property
     def dest_interval(self):
         return slice(0, self.zone_data.nr_zones)
+    
+    def add_destination_cost(self, mtx, vector):
+        """Add zonedata to matrix. """
+        mtx = numpy.add(mtx, vector)
+        return mtx
```

Comment:
I do not see the point in having this wrapper function, when you could simply write ` day_imp[mode][mtx_type] += park_cost` in `transform_impedance`.

Reply:
This is nice solution!